### PR TITLE
issues.md: update labels list #7595

### LINKED ISSUE #7595: issues.md: update labels list
The following labels were decommissioned:
- `c.Enhancement`, `c.Story`, `c.Task`

The following labels were added:
- `a-Storage`: database schema, GAE datastore
- `f-Questions`: features related to specific question types
- `f-Sessions`: feedback session creation, editing
- `f-Website`: front page, product website
- `t-Java`: Java-related technologies, e.g. language syntaxes

The following labels need to adapt in lieu of the added labels:
- `a-Persistence`: eventual consistency, CRUD operations
- `f-Submissions`: feedback submissions
- No `t-*` label: usually documentation update, or mixture of many languages

### PR DESCRIPTION
**Pull request by Surya-06**
Fixes #7595

Changed the old tags to new descriptions and removed the decommissioned tags. Also added some new tags.

### CODE REVIEW

**Comment by chowyb** [comment]
@Surya-06 thanks for your contribution! The changes made in this pull request seems okay. However, please address the comments left by the bot. If you are unsure about the conventions, you may wish to have a look at [this page](https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#step-4-submit-a-pr).

**Comment by whipermr5** [comment]
Closing due to inactivity. @Surya-06 feel free to reopen once you've read and followed our guidelines :)

**Comment by Surya-06** [comment]
@whipermr5 Sorry for the delay , I will complete the requirements given by the bot ASAP.

**Comment by Surya-06** [comment]
@whipermr5  Could you please tell me how to change the PR description , I am new here and I can't seem t o find it anywhere ? :-)

**Comment by whipermr5** [comment]
@Surya-06 Just click on the edit icon. Your PR description is currently still in the wrong format (take a look at the other PRs as examples). I'll definitely reopen this once you fix it :)

**Comment by Surya-06** [comment]
@whipermr5  I am sorry to be so naive but I am not able to edit the pr description. When I click the edit button on the commits tab, it only allows me to change the pr title and does not allow me to change the description below it . Please help me :-(

**Comment by whipermr5** [comment]
screen shot 2017-07-09 at 11 45 05 pm

**Comment by whipermr5** [comment]
Your PR title is still wrong. Please follow [this](https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#step-4-submit-a-pr) **exactly**.

**Comment by Surya-06** [comment]
Is it ok now ?? Sorry for the inconvenience... @whipermr5

**Comment by Surya-06** [comment]
How do I remove the greyed out Fix number from the pr title ,  I can't seem to edit it.

**Comment by whipermr5** [comment]
Nope, still wrong. Refer to any of the merged PRs as examples.

> How do I remove the greyed out Fix number from the pr title

That's not part of the title. It isn't checked.

**Comment by Surya-06** [comment]
Ok , will do

**Comment by whipermr5** [comment]
Hint: if the PR title is correct, I should be able to tell at a glance which issue you're fixing.

**Comment by Surya-06** [comment]
hi

**Comment by Surya-06** [comment]
I am not able to get the title of the issue , the only lead I have is the issue number .
Could you give me the format of the pr title to be kept , the issue number is #7638

**Comment by whipermr5** [comment]
That's not the issue number; that's the PR number (of this PR). Have you read [docs/process.md#step-4-submit-a-pr](https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#step-4-submit-a-pr)?

**Comment by Surya-06** [comment]
I am sorry , I didn't know that . Where can I find the original issue so that I can get its issue number. Sorry for being a noob.

**Comment by whipermr5** [comment]
On the issues tab. You commented on the original issue about 10 days ago.

**Comment by whipermr5** [comment]
Hint: search for `issues.md` in the issue tracker, since that is what you changed...

**Comment by Surya-06** [comment]
Tq I'll get back soon

On Jul 10, 2017 8:33 AM, "John Yong" <notifications@github.com> wrote:

Hint: search for issues.md in the issue tracker, since that is what you
changed...

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/TEAMMATES/teammates/pull/7638#issuecomment-313990499>,
or mute the thread

**Comment by Surya-06** [comment]
@whipermr5 Thanks for the help , searched in the issue tracker and found the issue number to be #7595  . I have kept the same number in the title of the pr . Guess it solves the issue . Let me know if anything else is wrong .

**Comment by whipermr5** [comment]
Getting there :) Title and PR description are both still wrong though.

**Comment by whipermr5** [comment]
Follow points 2 and 3 of [docs/process.md#step-4-submit-a-pr](https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#step-4-submit-a-pr).

**Comment by Surya-06** [comment]
Just another minute.

**Comment by Surya-06** [comment]
Guess the title is done now , isn't it .

**Comment by Surya-06** [comment]
Is it done???? :-)

**Comment by Surya-06** [comment]
@whipermr5  Thanks for the support all the way through this , this was my first contribution and thanks for helping me out.  Could you also tell me how can I be of any further help?

Thanks a lot.

BTW Is there anything wrong again?

File: docs/issues.md [hunk 1]
```diff
@@ -68,10 +65,11 @@ This label classifies the issue based on the **non-functional aspect** it tackle
 * `a-Docs`: website, user docs, dev docs
 * `a-FaultTolerance`: resilience to user errors, environmental problems
 * `a-Performance`: speed of operation
-* `a-Persistence`: database layer, GAE datastore
+* `a-Persistence`: eventual consistency, CRUD operations
 * `a-Process`: workflow management, release management
 * `a-Scalability`: related to behavior at increasing loads
 * `a-Security`: protection from security threats
+* `a-Storage`: database store, GAE datastore
```
Comment (whipermr5) [inline docs/issues.md]: From the original issue, this should be `database schema, GAE datastore`

Comment (Surya-06) [inline docs/issues.md]: Can I edit the file directly from the commits tab ??

Comment (whipermr5) [inline docs/issues.md]: I think so; try it!

**Comment by Surya-06** [comment]
@whipermr5  Sorry for the mistake , I have changed the value . Hope its ok now .

**Comment by Surya-06** [comment]
@whipermr5  Thanks a lot !!!!

**Pull request merged**